### PR TITLE
Update testing instructions and remove mentions of unused tech

### PR DESCRIPTION
**Pull request by andrablaj**
# Description

Update testing instructions and remove mentions of unused tech (e.g. `redis`, `logstash` & `postgrest`).

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+Pre-requisites:
+* `Node.js` version > 20
 
-2. Install the dependencies via:
+In order to run these end-to-end tests locally, follow the steps below:
+
+1. Install the dependencies via:
 
 ```sh
 npm install
```
Comment (dianabarsan) [inline TESTING.md:12]: This should be `npm ci` always!

File: TESTING.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 ```
 
-3. Run the Docker containers locally:
+2. Install the `couch2pg` dependencies via:
```
Comment (dianabarsan) [inline TESTING.md:15]: Do you think we should make this a post install hook?

File: TESTING.md [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+
 ```sh
-# starts: logstash, postgres, postgrest, data-generator, couchdb and dbt
-docker-compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f docker-compose.yml up -d
+cd couch2pg && npm install
```
Comment (dianabarsan) [inline TESTING.md]: `npm ci`

**Review by dianabarsan — APPROVED** [review]
Just minor comments. Thanks for taking this on!

**Review by dianabarsan — APPROVED** [review]

File: TESTING.md (hunk 2 above)
Comment (andrablaj) [reply TESTING.md:15]: Good point! I don't see any drawback, and removing a manual step is a small win. I will add another commit shortly to address this.

Comment (andrablaj) [reply TESTING.md]: Done

File: TESTING.md [hunk 4]
```diff
@@ -1,22 +1,18 @@
...
-docker-compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f docker-compose.yml up -d
+npm ci
 ```
 
-3. Wait for every container to be up and running.
-4. Run the end-to-end tests:
+2. Run the end-to-end tests:
 
 ```sh
 # runs tests/e2e-test.spec.ts
```
Comment (dianabarsan) [inline TESTING.md:18]: I _think_ the command to run e2e tests is `npm run test:e2e`. `npm test` right now only runs unit tests for couch2pg (and in the future, should run any unit tests that exist in the monorepo).

**Review by dianabarsan — APPROVED** [review]

File: TESTING.md (hunk 4 above)
Comment (andrablaj) [reply TESTING.md]: You are perfectly right; thanks for pointing it out. I separated the instructions into running unit tests & running e2e tests.

**Pull request merged**